core/emulate.c

```diff
@@ -772,6 +859,22 @@ static em_status_t decode_op_vex_reg(em_context_t *ctxt,
     op->type = OP_REG;
     op->size = ctxt->operand_size;
     op->reg.index = ~ctxt->vex.v & 0xF;
+    op->reg.shift = 0;
+    return EM_CONTINUE;
+}
+
+static em_status_t decode_op_opc_reg(em_context_t *ctxt,
+                                     em_operand_t *op)
+{
+    op->type = OP_REG;
+    op->size = ctxt->operand_size;
+    op->reg.index = ctxt->b & 0x7 | (ctxt->rex.b << 3);
```

Comment:
This line fails the `haxm-darwin` Travis CI build. Obviously, clang requires us to write `(ctxt->b & 0x7)`:

```
/Users/travis/build/intel/haxm/core/emulate.c:871:29: error: '&' within '|' [-Werror,-Wbitwise-op-parentheses]
    op->reg.index = ctxt->b & 0x7 | (ctxt->rex.b << 3);
                    ~~~~~~~~^~~~~ ~
/Users/travis/build/intel/haxm/core/emulate.c:871:29: note: place parentheses around the '&' expression to silence this warning
    op->reg.index = ctxt->b & 0x7 | (ctxt->rex.b << 3);
                            ^
                    (            )
1 error generated.
```